### PR TITLE
Adding fosdem 2017 event to opensourcedesign.net

### PR DESCRIPTION
**Pull request by ei8fdb**
This is a duplicate of the FOSDEM 2016 event. I’ve dated this for 10
Oct 2016.

I guess once the organisers have decided to accept our room we can push it merge it….

### CODE REVIEW

**Comment by evalica** [comment]
Hi :) I guess I should have looked at the PRs first and then create a new one. 

Anyway, there is also https://github.com/opensourcedesign/events/pull/46 

Still I don't understand why there are 3 events in this PR.

File: _posts/2016-04-15-libregraphicsmeeting.markdown [hunk 1]
```diff
@@ -0,0 +1,21 @@
+---
+layout: post
+title:  "Libre Graphics Meeting"
+date:   2016-04-15
+categories: libregraphics meeting
+eventDate: Tue, 15 April 2016 – Sat, 18 April 2016
+location: London, UK
+time: full days
+status: future
```
Comment (jancborchardt) [inline _posts/2016-04-15-libregraphicsmeeting.markdown:9]: We already have this event. :)

File: _posts/2016-10-10-fosdem.markdown [hunk 2]
```diff
@@ -0,0 +1,27 @@
+---
+layout: post
+title:  "Fosdem 2017"
```
Comment (jancborchardt) [inline _posts/2016-10-10-fosdem.markdown:3]: We already have this through #46 as @evalica said :) can you remove the file?

File: _posts/2017-03-06-internetfreedomfestival.markdown [hunk 3]
```diff
@@ -0,0 +1,29 @@
...
+categories: design internet-freedom activism
+eventDate: Tue 6 - Fri 10 March 2017
+location: Calle de Juan Verdeguer 16, 46024 – Valencia, Spain
+time: All day
+status: upcoming
+---
+
+[The Internet Freedom Festival](https://internetfreedomfestival.org/) creates a common space where diverse communities working against and affected by censorship and surveillance can come together to teach, plan and act.
+
+The [Communications and Design theme](https://internetfreedomfestival.org/#themes) will connect designers, users, and developers to keep the tools secure and usable.
```
Comment (jancborchardt) [inline _posts/2017-03-06-internetfreedomfestival.markdown:14]: Thanks for adding this! :)

**Review by jancborchardt — CHANGES_REQUESTED** [review]
There’s too many events in this pull request @ei8fdb ;) and as of now, only one (the Internet Freedom Festival one) is not in the events list.

So I cherry-picked your commit with that one and put it in a pull request at https://github.com/opensourcedesign/events/pull/61 – please review :)

**Pull request closed without merging**